### DISCUSSION TITLE
Can't run in admin mode.

### DISCUSSION THREAD

**Discussion by FlamBy37** [mathoudebine/turing-smart-screen-python#579] [Q&A]
Hello there, i'm admin on my computer, but i don't know how to restart as admin. If someone can help me great thx
I search on github for find issue and don't have .
![Sans titre](https://github.com/user-attachments/assets/d3e7dcea-fbce-454f-8dc0-f1340e185f5b)

**Comment by c0ldJS** [comment]
When starting a command line session, are you telling Windows to `Run as administrator`?

**Reply by FlamBy37** [reply]
yes i did.
